### PR TITLE
Improve Timer test reliability part deux

### PR DESCRIPTION
**Pull request by townsend2010**
Adds a getter function for current Timer state for better testing Timer state expectations.
Relax timeouts in some tests to improve reliability.

### CODE REVIEW

File: src/utils/timer.cpp [hunk 1]
```diff
@@ -94,4 +93,9 @@ void Timer::stop()
         t.join();
 }
 
-} // namespace multipass::utils
+mpu::TimerState mpu::Timer::current_state()
```
Comment (Saviq) [inline src/utils/timer.cpp:96]:
```suggestion
mpu::TimerState mpu::Timer::state()
```

File: include/multipass/timer.h [hunk 2]
```diff
@@ -47,12 +47,13 @@ class Timer
     void pause();
     void resume();
     void stop();
+    TimerState current_state();
```
Comment (Saviq) [inline include/multipass/timer.h:50]:
```suggestion
    TimerState state();
```

**Review by Saviq — COMMENTED** [review]
I'd rename `current_state` to `state` on the public side of the interface and rename the private member to something else.

Otherwise, seems legit!

File: src/utils/timer.cpp (hunk 1 above)
Comment (townsend2010) [reply src/utils/timer.cpp]: I committed in my branch to avoid the cla check failure.

File: include/multipass/timer.h (hunk 2 above)
Comment (townsend2010) [reply include/multipass/timer.h]: I committed in my branch to avoid the cla check failure.

**Comment by townsend2010** [comment]
bors r=saviq,ricab

**Review by ricab — APPROVED** [review]
+0.5 (partial review: of the actual timer code, tests skipped).

**Comment by townsend2010** [comment]
Oh, good grief.  Try again:

bors retry

**Pull request merged**